source/includes/api/tables/progress-response.rst

```diff
@@ -81,3 +81,17 @@
      - If an error occurred, gives a detailed description of the error.
        This field is omitted when the call to the endpoint is successful
 
+   * - ``mongosyncID``
+     - string
+     - The identifier string for the ``mongosync`` instance.
+       To configure this value, see the :setting:`id` setting.
+
+   * - ``coordinatorID``
+     - string
+     - The identifier string for the coordinator instance.  To change this value,
+       see the :setting:`id` setting on the coordinator instance of ``mongosync``.
+
+       When the ``mongosync`` instance is the coordinator, this field returns
+       ``null``.
```

Comment:
This is incorrect, when the mongosync instance is the coordinator, the values of mongosyncID and coordinatorID will be equal.

coordinatorID could be null during mongosync start-up.

Reply:
Fixed